### PR TITLE
Use sensible defaults for non-secret configuration values. Also tidy …

### PR DESCRIPTION
**Pull request by dawngerpony**
…up the Dockerfile.

### CODE REVIEW

**Comment by george-angel** [comment]
As you are modifying `Dockerfile` could we please "delayer" it?

**Comment by dawngerpony** [comment]
@Zapadlo - could we do what now?

**Comment by dawngerpony** [comment]
@Zapadlo - please re-review.

File: Dockerfile [hunk 1]
```diff
@@ -1,49 +1,19 @@
     FROM alpine
```
Comment (george-angel) [inline Dockerfile]: fmt

File: Dockerfile [hunk 2]
```diff
@@ -1,49 +1,19 @@
...
-#  && go get \
-#  && go test \
-#  && go build ${REPO_PATH} \
-#  && apk del go git alpine-sdk linux-headers \
-#  && rm -rf $GOPATH /var/cache/apk/*
+    && go build ${REPO_PATH} \
+    && apk del go git alpine-sdk linux-headers \
+    && rm -rf $GOPATH /var/cache/apk/*
 
 CMD ./coco-neo4j-backup
```
Comment (george-angel) [inline Dockerfile]: Could you please make this: `CMD ["./coco-neo4j-backup"]`

https://docs.docker.com/engine/reference/builder/#cmd

> `CMD ["executable","param1","param2"] (exec form, this is the preferred form)`

Comment (dawngerpony) [inline Dockerfile]: Done.

**Pull request merged**